lib/restful_model_collection.rb

```diff
@@ -37,6 +37,9 @@ def count
     end
 
     def first
+      @filters[:offset] = 0
```

Comment:
I'm not sure this is solving the completely. Isn't the root cause that the filters were shared by all subclasses of `RestfulModelCollection`?